app.js

```diff
@@ -57,18 +57,21 @@ switch(value) {
     case "navbar":
         activeTab(componentList,arrayOfListItems[10])
         break;
-    case "ratings":
+    case "modal":
         activeTab(componentList,arrayOfListItems[11])
         break;
-    case "toast":
+    case "ratings":
         activeTab(componentList,arrayOfListItems[12])
         break;
-    case "grid":
+    case "toast":
         activeTab(componentList,arrayOfListItems[13])
         break;
-    case "slider":
+    case "grid":
         activeTab(componentList,arrayOfListItems[14])
         break;
+    case "slider":
+        activeTab(componentList,arrayOfListItems[15])
```

Comment:
you can use a hash map object to read the array variable. `let listItems={ toast:"",grid:"", ...}`
while reading the variable `listItems['toast']` etc. This is more readable, and in case your array list scales up, it
s easier to modify changes than recalling which index is relevant to which item.

Reply:
Done with changes